pipelines/frontend-deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: frontend-pipeline
+name: frontend-deploy
```

Comment:
Same as the api-deploy, how would we add a dependency to only trigger deployment after a build had succeeded?

Reply:
See answer above;  TLDR: based on branch + path policies.  Notice the "build" pipelines don't have branch specified but do specify the "PR" section meaning it will run on all PRs.